CHANGELOG.md

```diff
@@ -12,6 +12,120 @@
+- Add new bypassValidation query param to Add/Patch Device API ([#4797](https://github.com/edgexfoundry/edgex-go/issues/4797)) ([17930b1…](https://github.com/edgexfoundry/edgex-go/commit/17930b14352cbfb7930b1ac1271c629a4bf190e4))
+- Add OpenZiti to edgex-go ([#4777](https://github.com/edgexfoundry/edgex-go/issues/4777)) ([d2d9888…](https://github.com/edgexfoundry/edgex-go/commit/d2d98883e5139249b4399f4dc0dd90a078163ffc))
+- Allow sending notifications to authenticated EdgeX endpoints ([#4763](https://github.com/edgexfoundry/edgex-go/issues/4763)) ([a48d639…](https://github.com/edgexfoundry/edgex-go/commit/a48d6392372de4eada0148d46cda0c5092cdf95e))
+- Add new StoreForwardQueueSize metric to common App services config ([#4751](https://github.com/edgexfoundry/edgex-go/issues/4751)) ([3a892ac…](https://github.com/edgexfoundry/edgex-go/commit/3a892ac0b38b8d685f9d63feb7dde70ee06d273b))
+
+- Add core-metadata device/provisionwatcher db methods for postgres ([fd73939…](https://github.com/edgexfoundry/edgex-go/commit/fd73939b4ae038ef84b6f1c31c78656397dfc769))
+```text
+
+BREAKING CHANGE: Switched default database to PostgreSQL across all services
+
+```
+
+### Code Refactoring ♻
+- Refactor services to build monolithic service easily ([#5010](https://github.com/edgexfoundry/edgex-go/issues/5010)) ([98bb6eb…](https://github.com/edgexfoundry/edgex-go/commit/98bb6ebf5936140bbb14ec64c05520fd1477cb0f))
+
+- Replace old Scheduler service with cron Scheduler service ([505c688…](https://github.com/edgexfoundry/edgex-go/commit/505c688739c6baeb7936588b5c612c7d4ee74cdd))
+```text
+
+BREAKING CHANGE: Refactored old Scheduler service to support advanced job schedulin
+
+```
+- Replace vault with openbao ([2fd20ad…](https://github.com/edgexfoundry/edgex-go/commit/2fd20adf2de51a59e2f08b9fbb81467a4bed6eac))
+```text
+
+BREAKING CHANGE: replace Vault with OpenBao
+
+```
+- Update module to v4 ([da73523…](https://github.com/edgexfoundry/edgex-go/commit/da735230bec8d3079658cac996101c0c1901d17e))
+```text
+
+BREAKING CHANGE: update go module to v4
+
+```
+<!-- not sure -->
+<!-- - Separate count and query functions in DB interface ([803138e…](https://github.com/edgexfoundry/edgex-go/commit/803138eab89292506fff25623a654a487eec4213)) -->
+
+
+### Bug Fixes 🐛
+
+- Fix multiple bugs in support-scheduler service ([9100ac2…](https://github.com/edgexfoundry/edgex-go/commit/9100ac240e6a97fe5a84fed5dc78596f5db1e64b))
+- Fix PostgreSQL-specific issues like permission errors, invalid limits, and race conditions ([3f02c68…](https://github.com/edgexfoundry/edgex-go/commit/3f02c68d3e0b5122ed047bfb78be384dc5192cd3))
+- Check offset, limit, totalCount before querying transmissions ([82df6aa…](https://github.com/edgexfoundry/edgex-go/commit/82df6aa76d7e5e8e8165c23ce0e8560af0818afb))
+- Only one ldflags flag is allowed ([fcb5d6c…](https://github.com/edgexfoundry/edgex-go/commit/fcb5d6c99dbec038e1b3b8f6405170469af5ccc2))
```

Comment:
remove this because `Add ldflags to strip debug info from binaries` in the build section is enough